README.md

```diff
@@ -99,7 +99,26 @@ An options object can be passed as the second parameter to `cronstrue.toString`.
 
 ## i18n
 
-To use the i18n support cRonstrue provides, you can import a specific locale and then call `toString()`. For example, for the es (Spanish) locale:
+cRonstrue provides a few ways to use i18n, depending on your situation.
+
+1. Use the packaged library that contains the locale translations.  
+Once you do this, you can pass the name of a supported locale as an option to  `cronstrue.toString()`.  
+For example, for the es (Spanish) locale, you would use: `cronstrue.toString("* * * * *", { locale: "es" });`.  
+
+```js
+var cronstrue = require('cronstrue/i18n');
+cronstrue.toString("*/5 * * * *", { locale: "fr" });
+```
+
+2. Import the specific locales you want:  
+
+```js
+var cronstrue = require('cronstrue');
+require('cronstrue/locales/fr');
+cronstrue.toString("*/5 * * * *", { locale: "fr" });
+```
+
+3. Directly import the locale and call its `toString`. For example, for the es (Spanish) locale:
```

Comment:
Wouldn't this method no longer work since you're marking cronstrue as external in the Webpack config?  I think you would _have_ to use method 2, where you first import cronstrue, yes?

Reply:
If you require it, it requires cronstrue, and it works fine :)

Only diff is that for browser you'll have to include cronstrue first. In node/webpack you dont have to.